reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,69 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is a distinct set of tools and components that ships separately from Windows PowerShell.
+Therefore, PowerShell Core is not included in the Windows 10 or Windows Server licensing agreements.
+
+However, PowerShell Core is supported under traditional Microsoft support agreements, including [Premier][] and [Microsoft Enterprise Agreements][enterprise-agreement].
+You can also pay for [assisted support][] for PowerShell Core by filing a support request for your problem.
+
+We also offer [community support][] on GitHub where you can file an issue, bug, or feature request.
+We offer no guarantee there that your issue will be addressed or resolved in a timely manner.
```

Comment:
However, we offer...

Reply:
Should we mention MS Forums?
Should we mention UserVoice?

Reply:
UserVoice, I'd rather not...

MS forums, which ones are the kids using nowadays? I think TechNet isn't in widespread use anymore, right? Do people use this? https://answers.microsoft.com/en-us